config/database.yml

```diff
@@ -5,7 +5,7 @@ postgresql: &postgresql
   port: 5432
 
 defaults: &defaults
-  pool: 5
+  pool: <%= (File.exist?('/etc/login.gov/info') && File.read('/etc/login.gov/info/role').chomp == 'worker') ? 26 : 5 %>
```

Comment:
Thoughts on setting this for just the prod environment, and not the default environment?

Reply:
We need this for other ends too (pt and sometimes staging)

Reply:
I think what Jonathan meant by environment was "development", "test", and "production". All the lower envs are running in the Rails "production" environment. So, he's asking to make the change in the "production" section at the bottom of this file.

Reply:
ahh, that makes sense. Sorry for misunderstanding @jmhooper and thanks for clueing me in @monfresh!